cli/src/utils.rs

```diff
@@ -246,10 +246,12 @@ pub(crate) use p_println;
 /// Disables terminal colours if either:
 /// - Running windows and the terminal does not support colour codes.
 /// - Colour has been disabled by some environment variable.
+/// - We are running inside a test
 pub fn enable_paint() {
+    let is_test = cfg!(test);
     let is_windows = cfg!(windows) && !Paint::enable_windows_ascii();
     let env_colour_disabled = std::env::var("NO_COLOR").is_ok();
-    if is_windows || env_colour_disabled {
+    if is_test || is_windows || env_colour_disabled {
```

Comment:
Windows doesn't support color out of the box so instead of replacing every ANSI escape code in our fixtures I just opted to disable colors in tests. I think this is OK, if the colors are off it's not critical and we'd know easily anyway